### ISSUE TITLE
Publish unminified source code instead of CRX/XPI bundles

### ISSUE THREAD

**Issue by noahbetzen-wk** [shridhar-tl/jira-assistant#68]
Right now this repo only publishes CRX/XPI bundles for Chrome/Firefox. This makes auditing the source code difficult because you first have to use a tool like: https://crxextractor.com/

Even after extracting the bundles, the code is minified and unreadable.

**Comment by seanericwatson** [comment]
@noahbetzen-wk Have you tried this one: https://chrome.google.com/webstore/detail/chrome-extension-source-v/jifpbeccnghkjeaalbbjmodiffmgedin?hl=en

**Comment by noahbetzen-wk** [comment]
Even using an extension like that still produces minified code. It's another layer of obfuscation. The original unminified code should be public by default and easy-to-access for everyone.

**Comment by Regenhardt** [comment]
Using tools to revert the obfuscation isnt a solution. 
I'd actually like to help development on this, which isnt possible without the full source either.

**Comment by nitram509** [comment]
Same from my perspective ... having some ideas for improving project planning via Gantt Chart like view to contribute.
Is there a chance you to open the sources, @shridhar-tl ?

**Comment by shridhar-tl** [comment]
Hi All,
CC: @noahbetzen-wk @seanericwatson @Regenhardt @nitram509 

Sorry, I was quiet busy with my official activity since few months and was not able to contribute for this project in anyway. After getting lots of such request through multiple medium, I had decided to publish it as opensource project here in GitHub.

But current code base has lots of issues and it would be difficult for others to work on. Also the size of the build package has grown up which also cause performance issues.

So since a month, I was in progress to rewrite the entire application in React. It is not yet ready to be published, but is currently in state where others can contribute. This weekend I would publish it and from next Monday onwards whoever can contribute are very much Welcome to do so.

**Comment by nitram509** [comment]
Awesome 👍 
I'm looking forward to ...

**Comment by shridhar-tl** [comment]
Latest code is now available in develop branch. You can have a look at it and anyone before starting to work on anything please let me know. Better create a new issue with all the required details and proceed to work with it after I reply to it. Please keep in mind that,any changes you make should work in all the browser and should be generic to all users. No implementation is allowed which is specific to your requirements. Soon I will try to create a document with some guidelines for others who is trying to contribute in development.

Thank you all! Waiting for better working Jira Assistant!!

**Comment by nitram509** [comment]
👍

**Comment by nitram509** [comment]
This ticket can be closed then :)

**Comment by shridhar-tl** [comment]
Closed as the source code is now available in develop branch and anyone willing to contribute can do so.

**Comment by seanericwatson** [comment]
Fantastic news! Thank you @shridhar-tl , my whole team is excited about starting to look at the code and figure out how we might be of contribution. We'll prepare some ideas about how we might work with the project for your review and approval.

Warm regards,
Sean
